assets/rbac/main_provisioner_binding.yaml

```diff
@@ -1,12 +1,12 @@
 kind: ClusterRoleBinding
 apiVersion: rbac.authorization.k8s.io/v1
 metadata:
-  name: manila-csi-provisioner-binding
+  name: manila-csi-main-provisioner-binding
 subjects:
   - kind: ServiceAccount
     name: manila-csi-driver-controller-sa
     namespace: openshift-manila-csi-driver
 roleRef:
   kind: ClusterRole
-  name: manila-external-provisioner-role
```

Comment:
Incidentally, does something somewhere handle cleaning up the old roles?

Reply:
Thank you for review, @mdbooth , appreciated!

This PR cleaned up old roles. E.g, it deleted `assets/rbac/provisioner_role.yaml` which published `manila-external-provisioner-role`.  This role was specific for manila operator, so with this PR applied we don't have either `manila-external-provisioner-role` or corresponding binding.